beam/core/src/main/java/cz/o2/proxima/beam/direct/io/BlockingQueueLogObserver.java

```diff
@@ -63,6 +64,7 @@ private BlockingQueueLogObserver(String name, long limit, long startingWatermark
     this.watermark = new AtomicLong(startingWatermark);
     this.limit = limit;
     queue = new ArrayBlockingQueue<>(100);
+    log.debug("Created {}", this);
```

Comment:
Can we have a better log message here?

Reply:
What do you miss? The intent was to log that a new BlockingQueueLogObserver was created.